docs/3.api/2.composables/use-google-tag-manager.md

```diff
@@ -0,0 +1,60 @@
+---
+title: useGoogleTagManager
+description: useGoogleTagManager allows you to install Google Tag Manager in your Nuxt app.
+links:
+  - label: Source
+    icon: i-simple-icons-github
+    to: https://github.com/nuxt/scripts-and-assets/blob/main/modules/nuxt-third-party-capital/src/runtime/composables/googleTagManager.ts
+    size: xs
+---
+
+The `useGoogleTagManager` composable function allows you to install [Google Tag Manager](https://developers.google.com/tag-platform/tag-manager/web) in your Nuxt application.
+
+## Type
+
+```ts
+useGoogleTagManager(options: ThirdPartyScriptOptions<GoogleTagManagerOptions, GoogleTagManagerApi>): ThirdPartyScriptApi<GoogleTagManagerApi>
+```
+
+## Params
+
+An object containing the following options:
+
+| name | type   | description                     |
+|:-----|:-------|:--------------------------------|
+| id   | string | Your GTM container ID. Usually starts with GTM-. (required) |
+
+## Return values
+
+An object that contains a special `$script` property that gives you access to the underlying script instance.
+
+- `$script.waitForLoad`: A promise that resolves when the script is ready to use. It exposes `google_tag_manager` and `dataLayer`, which lets you interact with the API.
+
+::callout
+Learn more about [useScript](https://unhead.unjs.io/usage/composables/use-script).
+::
+
+## Minimal example
```

Comment:
Happy to hear if you think otherwise, but I think it would be nicer to move the examples to earlier before the other sections (Type, Params, Return values)